087-single-step-multi-version-downgrade.md

```diff
@@ -0,0 +1,45 @@
+# Support Single step multi version downgrade for Zookeeper based clusters 
+
+This proposal seeks to introduce support for multi-version downgrade of Strimzi in a single step where possible.
+The proposal only relates to Zookeeper based clusters.
```

Comment:
As I commented on the https://github.com/strimzi/strimzi-kafka-operator/issues/10801 issue. This does not seem to make any sense anymore:
* At best, this would land in Strimzi 0.45 with support for Kafka 3.8 and 3.9.
* These are the last versions with ZooKeeper support. ZooKeeper will be dropped in Strimzi 0.46 / Kafka 4.0
* So there will be never a downgrade of a ZooKeeper-based cluster that would utilize this feature

Reply:
I agree, doesn't make sense for Zookeeper at this stage. I've updated the proposal to address the same issue for KRaft based clusters instead